src/pinghelper.cpp

```diff
@@ -49,6 +49,9 @@ void PingHelper::start(const QString& serverIpv4Gateway,
   if (!m_pingSender->isValid()) {
     m_pingSender->deleteLater();
     m_pingSender = new DnsPingSender(m_source, this);
+    if (!((DnsPingSender*)m_pingSender)->start()) {
+      logger.error() << "Unable to start DNSPingSender.";
```

Comment:
I've just made it a dummy ping sender for now, and log it. I agree it's better to bubble up further, but this seems like an improvement from what is currently there, and I'm not sure the amount of time it would take to bubble up further is worth it right now, given all the other priorities we have. Open to having my mind changed.